### ISSUE TITLE
R1-J5 did not trigger on direct hit repair

### ISSUE THREAD

**Issue by charlieneo** [Sandrem/FlyCasual#3081]
R1-J5 should be able to repair after direct hit resolve.

**Comment by rune-b** [comment]
R1-J5 currently only triggers on repair actions, not the automatic repair that happens for some cards

**Comment by charlieneo** [comment]
It shown that it can repair off an automatic repair from damage cards in the preview videos from FFG. Also in the FAQ there is a clarification about repairing after being destroyed which is possible only if its being triggered by a damage card auto repair.

Q: If a ship is destroyed and an effect such as R1-J5 [󲁔]
repairs one or more of its damage cards before it is
removed, is the ship still destroyed (and thus removed)?
A: Yes. After a ship becomes "destroyed" for any reason, it remains
destroyed no matter what effects are resolved before it is removed.
Effects can change the timing at which a ship is removed, but cannot undo
the state of being destroyed.

**Comment by rune-b** [comment]
Yes, I was not disagreeing with you, just writing down how it currently works in FlyCasual
